### ISSUE TITLE
Add clip iterations

### ISSUE THREAD

**Issue by shady-cs15** [shady-cs15/slash#6]
`clip_iter`=20
`n_epochs` = 10
If accuracy doesn't improve, use constant divisor for normalizing in stead of `tf.reduce_sum`.
